include/nnvm/top/nn.h

```diff
@@ -125,6 +125,9 @@ struct Conv2DParam : public dmlc::Parameter<Conv2DParam> {
   int groups;
   int layout;
   bool use_bias;
+  bool use_batchNorm;
```

Comment:
We should not change the conv2d definitions, instead, append a batch_norm operator after the conv2d during the conversion step